pkg/apis/jenkins.io/v1/types_environment.go

```diff
@@ -12,7 +12,15 @@ import (
 
 // +genclient
 // +genclient:noStatus
-// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
```

Comment:
This is deprecated, reference: https://book.kubebuilder.io/reference/markers/object.html